docs/source/index.rst

```diff
@@ -1,132 +1,89 @@
-.. _Modbus-Driver:
+.. _DNP3-Driver:
 
-=============
-Modbus Driver
-=============
+===========
+DNP3 Driver
+===========
 
+VOLTTRON's DNP3 driver enables the use of `DNP3 <https://en.wikipedia.org/wiki/DNP3>`_ (Distributed Network Protocol)
+communications, reading and writing points via a DNP3 Outstation.
 
-
-VOLTTRON's modbus driver supports the Modbus over TCP/IP protocol only. For Modbus RTU support, see VOLTTRON's
-`Modbus-TK driver <Modbus-TK-Driver>`.
-
-`About Modbus protocol <https://en.wikipedia.org/wiki/Modbus>`_
-
-
-.. _Modbus-Config:
-
-Modbus Driver Configuration
-===========================
+In order to use a DNP3 driver to read and write point data, a server component (i.e., Outstation) must also
+be configured and running.
 
 Requirements
-------------
-The Modbus driver requires the pymodbus package. This package can be installed in an activated environment with:
+============
 
-.. code-block:: bash
-
-    pip install pymodbus
-
-Alternatively this requirement can be installed using :ref:`bootstrap.py <Platform-Installation>` with the ``--drivers``
-option:
+The DNP3 driver requires the `dnp3-python <https://github.com/VOLTTRON/dnp3-python>`_ package, a wrapper on Pydnp3 package.
+This package can be installed in an activated environment with:
 
 .. code-block:: bash
 
-    python3 bootstrap.py --drivers
+    pip install dnp3-python
 
 
 Driver Configuration
---------------------
+====================
+
+There is one argument for the "driver_config" section of the DNP3 driver configuration file:
 
-There are three arguments for the `driver_config` section of the device configuration file:
+Here is a sample DNP3 driver configuration file:
 
-    - **device_address** - IP Address of the device.
-    - **port** - Port the device is listening on.  Defaults to 502 which is the standard port for Modbus devices.
-    - **slave_id** - Slave ID of the device. Defaults to 0.  Use 0 for no slave.
+.. code-block:: json
 
-The remaining values are as follows:
+    {
+      "driver_config": {
+        "master_ip": "0.0.0.0",
+        "outstation_ip": "127.0.0.1",
+        "master_id": 2,
+        "outstation_id": 1,
+        "port": 20000
+      },
+      "registry_config": "config://dnp3.csv",
+      "driver_type": "dnp3",
+      "interval": 5,
+      "timezone": "UTC",
+      "publish_depth_first_all": true,
+      "heart_beat_point": "random_bool"
+    }
 
+A sample DNP3 driver configuration file can be found in the VOLTTRON repository
+in `example-config/dnp3.config
+<https://github.com/eclipse-volttron/volttron-lib-dnp3-driver/blob/main/example-config/dnp3.config>`_.
 
 
-Here is an example device configuration file:
+DNP3 Registry Configuration File
+================================
 
-.. code-block:: json
+The driver's registry configuration file, a `CSV <https://en.wikipedia.org/wiki/Comma-separated_values>`_ file,
```

Comment:
This one isn't on you, since all the current documentation says something very similar, but based on feedback during the quarterly meeting, I'm going to be trying to make a little more clear in the documentation that we are not required to use CSV for registries. I would suggest the following here:

"The driver's registry configuration may be a CSV file with one row for each point or a JSON file strucutred as a list of objects (one object for each point). The following columns (CSV) or keys (JSON) are required for each point:"

You don't need to show the JSON version. Just indicate further down:
"Point definitions in the DNP3 driver’s registry (shown as CSV) should look similar to"